src/3D/Water.cpp

```diff
@@ -38,6 +40,13 @@ Water::Water()
 {
 	_reflectionFrameBuffer =
 	    std::make_unique<FrameBuffer>("Reflection", 1024, 1024, graphics::Format::RGBA8, graphics::Format::Depth24Stencil8);
+
+	// water texture
+	auto const& waterTextureData = Game::instance()->GetFileSystem().ReadAll("Data/Textures/Sky.raw");
+
+	_texture = std::make_unique<Texture2D>("Water");
+	_texture->Create(256, 256, 1, Format::RGB8, Wrapping::ClampEdge, waterTextureData.data(), waterTextureData.size());
```

Comment:
Can we name these values for width and height instead of plugging the special 256 number everywhere?